cmake/onnxruntime_visionos.toolchain.cmake

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set(CMAKE_SYSTEM_NAME visionOS)
+set(CMAKE_SYSTEM_PROCESSOR arm64)
```

Comment:
What would be used to build the simulator? Or is arm64 required for that as well?

Reply:
right, I think for xrsimulator(simulator for vision os) it would also require arm64 architecture